### ISSUE TITLE
[BUG] Error with UVs in the class cube.ts

### ISSUE THREAD

**Issue by arthurlebourg** [DavidPeicho/Teaching#4]
There is an error with the indices vector of the Cube which make the program unable to uses UV coordinates with a cube object.
The indices vector or the uvs vector is to blame: there are 24 indices asked in the indices vector but only 4 provided by uvs vector.

**Comment by DavidPeicho** [comment]
Good point. I guess I started doing it and forgot to finish. Do you want to fix it and make a PR for 0.25 points?
